.devcontainer/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# This Dockerfile is currently only used to enforce use of x86 image on ARM64 CPUs (Apple Silicon)
+# See aarch64 migration on https://conda-forge.org/status/, which is currently blocking us.
+FROM --platform=amd64 mcr.microsoft.com/devcontainers/base:debian-11
```

Comment:
enforcing AMD64 platform here

Reply:
Ubuntu 22 and Debian 12 ship with OpenSSL 3, which dropped support for some older, insecure algorithms.

Our corporate firewall still injects some of these older certs in some places, so certain encrypted connections fail to be established. Sucks but that's how it is currently, not just for me. 

All packages we need seem to be available in recent-enough versions. Is there anything particular you expect to be problematic? Remember we're not trying to provide an all-purpose dev environment here, only one specifically tailored for MNE development.

Reply:
Using Debian 12 now.